config/dev.exs

```diff
@@ -60,7 +60,7 @@ config :live_view_demo, LiveViewDemoWeb.Endpoint,
     patterns: [
       ~r"priv/static/.*(js|css|png|jpeg|jpg|gif|svg)$",
       ~r"priv/gettext/.*(po)$",
-      ~r"lib/live_view_demo_web/{live,views}/.*(ex)$",
+      ~r"lib/live_view_demo_web/(live|views)/.*(ex)$",
```

Comment:
This one was filed as a PR to the upstream demo project: https://github.com/DockYard/live_view_demo/pulls